pages/search.js

```diff
@@ -202,14 +189,33 @@ export default function Search({ data: { users, tags, randUsers } }) {
           />
         </Badge>
 
-        {notFound && <Alert type="error" message={`${notFound} not found`} />}
+        {notFound && <Alert type="error" message={`${notFound}`} />}
```

Comment:
I think this can be done like so

```suggestion
        {notFound && <Alert type="error" message={notFound} />}
```